tests/test_helpers.py

```diff
@@ -32,6 +32,35 @@
       "--authors", "sbillinge", "ashaaban", "--abstract", "the earth is round as seen from mars",
       "--title", "On the roundness of the Earth", "--status", "in-prep"],
      "2006as_mars has been added in presentations\n"),
+    (["helper", "l_progress", "-l", "ascopatz"],
+     "*************************[Proposed Projecta]*************************\n"
+      "20sb_firstprojectum\n"
+      "    status: proposed, begin_date: 2020-04-28, due_date: 2021-05-05\n"
+      "    description: My first projectum\n"
+     ),
+    (["helper", "l_progress", "-v", "-l", "ascopatz"],
+     "*************************[Proposed Projecta]*************************\n"
+      "20sb_firstprojectum\n"
+      "    status: proposed, begin_date: 2020-04-28, due_date: 2021-05-05\n"
+      "    description: My first projectum\n"
+      "    log_url: https://docs.google.com/document/d/1YC_wtW5Q\n"
+      "    team:\n"
+      "        group_members: ascopatz\n"
+      "        collaborators: aeinstein, pdirac\n"
+      "    deliverable:\n"
+      "        due_date: 2021-05-05\n"
+      "        audience: beginning grad in chemistry\n"
+      "        scope: 1. UCs that are supported or some other scope description if it is software\n"
+      "               2. sketch of science story if it is paper\n"
+      "        platform: description of how and where the audience will access the deliverable.  Journal if it is a paper\n"
+      "    milestones:\n"
+      "        2020-05-20: Project lead presentation\n"
+      "            objective: lead presents background reading and initial project plan\n"
```

Comment:
I think it is ok to have objective here, but I did want a notes field.  It is not a required database entry but it is allowed and it is a place where we can dump notes to return to later.  Please check the schema to make sure I am right on this.  But the point of the notes is to return to them later if we need, so it is important that any notes are reproduced in the report.  Actually, I think there can be a prum-level notes and a milestone level notes so if notes is found a the the prum level it should appear somewhere up higher, maybe just before deliverable.  Mostly there won't be notes in which case just don't print anything for notes.  In the  test you may have to add a couple of notes field to one of the projecta in the schema to test it.

Reply:
I have checked again - there is no "notes" field allowed for "projecta." I tried to add a "notes" field in schema, and then many tests failed. There can be notes for many other items, such as "proposals", "presentations", etc. Please let me know if I am wrong.